backend/src/lib/server-side-events.ts

```diff
@@ -277,12 +278,24 @@ export class ServerSideEvents {
     }
 
     let sentCount = 0
+    let allEvents: ServerSideEvent[] = []
     for (const eventID in this.events) {
       if (Number(eventID) <= lastEventID) continue
-      this.sendEvent(clientID, this.events[eventID])
-      sentCount++
+      allEvents.push(this.events[eventID])
+    }
+
+    // mock a large environment
+    if (process.env.MOCK_CLUSTERS) {
```

Comment:
This is for test right?

Reply:
Retaining test code is good!

Reply:
I removed that--- I had added functionality to mock lots of clusters but I moved it into it's own pr